chia/full_node/mempool_check_conditions.py

```diff
@@ -107,16 +108,38 @@ def add_cond(
     conds[op].append(ConditionWithArgs(op, args))
 
 
+def unwrap(x: Optional[uint32]) -> uint32:
+    assert x is not None
```

Comment:
`assert` can get optimized out if you go `python -O` (what a stupid idea) so you should consider `if x is None: raise ValueError`

Reply:
yeah, this was the best I could think of to satisfy mypy. the value is known to not be `None` at this point